rust/crd/src/user_info_fetcher.rs

```diff
@@ -88,6 +94,27 @@ fn aas_default_port() -> u16 {
     5000
 }
 
+#[derive(Clone, Debug, Deserialize, Eq, JsonSchema, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ActiveDirectoryBackend {
+    /// Hostname of the domain controller, e.g. `ad-ds-1.contoso.com`.
+    pub ldap_hostname: String,
```

Comment:
In the AuthenticationClass we also have an `ldapPort` field: https://docs.stackable.tech/home/stable/concepts/authentication#ldap. It might make sense to have that for consistency reasons as well.

If not please test a non-default port and update the CRD docs, e.g.
```
Hostname of the domain controller, e.g. `ad-ds-1.contoso.com` or `ad-ds-1.contoso.com:1234`.
```

Reply:
That's fair, the inspiration here was secret-op's LDAP support, which also doesn't support custom LDAP ports at the moment. As far as I know, there's no way to customize AD's LDAP port.

Reply:
In fairness, there are a few things we could port over from secret-op here:

- rename to `ldapServer`
- explicitly forbid port numbers (though it should break the SASL bind anyway)

Reply:
> As far as I know, there's no way to customize AD's LDAP port.

Oh that's a bit unexpected :D Happy to rename it to `ldapServer` and close this discussion

Reply:
94febb5bcec447bdcd80041585743e20b7cb4883